pkgs/applications/science/misc/graphia/default.nix

```
@@ -0,0 +1,30 @@
+{ stdenv, lib, cmake, fetchFromGitHub, qt5 }:
+
+let
+  version = "2.2";
+
+  src = fetchFromGitHub {
+    owner = "graphia-app";
+    repo = "graphia";
+    rev = "${version}";
+    sha256 = "sha256:05givvvg743sawqy2vhljkfgn5v1s907sflsnsv11ddx6x51na1w";
+  };
+
+in
+stdenv.mkDerivation {
+  pname = "graphia";
+  inherit version;
+
+  inherit src;
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ qt5.qtbase qt5.wrapQtAppsHook ];
+
+  meta = with lib; {
+    description = "A visualisation tool for the creation and analysis of graphs.";
+    homepage = "https://graphia.org";
+    license = licenses.gpl3;
+    maintainers = [ maintainers.bgamari ];
+    platforms = [ "x86_64-linux" ];
```

```suggestion
    platforms = platforms.all;
```

upstream supports darwin, linux and windows.